webpack/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "lazysizes",
+  "version": "0.0.1",
+  "main": "app/lazysizes.js",
+  "scripts": {
+    "build": "./node_modules/.bin/webpack -p",
+    "debug": "./node_modules/.bin/webpack --watch",
+    "watch": "./node_modules/.bin/webpack -p --watch"
+  },
+  "repository": {},
+  "license": "GPL-2.0",
+  "devDependencies": {
+    "babel-core": "^6.22.1",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.22.0",
+    "file-loader": "^1.1.5",
+    "image-webpack-loader": "^3.1.0",
+    "lazysizes": "^4.0.1",
```

Comment:
you should pin `4.0.1` or it could download a new version in the future.

Reply:
Done